### PR TITLE
UnicornHD driver implementation

### PR DESCRIPTION
**Pull request by jdortiz**
Implements the new display.Drawer, enforced at compile time via type assertion.
Uses physic.MegaHertz.
Includes new tests.

### CODE REVIEW

File: experimental/devices/unicornhd/unicornhd.go [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+	"image/color"
+	"image/draw"
+
+	"periph.io/x/periph/conn/display"
+	"periph.io/x/periph/conn/physic"
+	"periph.io/x/periph/conn/spi"
+)
+
+const (
+	Height = 16
```
Comment (maruel) [inline experimental/devices/unicornhd/unicornhd.go]: Why export?

File: experimental/devices/unicornhd/unicornhd_test.go [hunk 2]
```diff
@@ -0,0 +1,362 @@
+package unicornhd
```
Comment (maruel) [inline experimental/devices/unicornhd/unicornhd_test.go]: Include copyright here too.

**Comment by jdortiz** [comment]
Make width and height internal.

- Add copyright to tests.

**Comment by maruel** [comment]
gohci

**Comment by maruel** [comment]
gohci

**Pull request merged**